notebooks/QA_evaluation_metrics.ipynb

```diff
@@ -0,0 +1,330 @@
+{
```

Comment:
<p>Looks cool! </p><p>Minor suggestions: </p><ul><li>We could take an example from ROSA docs maybe? </li><li>And add LLM based metrics if they are reasonably easy to add?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Will update this soon!! Thanks Shrey.</p>